### PR TITLE
Event endpoint

### PR DESCRIPTION
**Pull request by jakeheft**
Description:  Add yelp endpoint to pull local events happening, update serializer to pass on different json data than for businesses

Fixes #

* [x] All tests passing
* [x] Lint code
* [x] Checked local host
* [x] Updated README

### CODE REVIEW

**Review by BJSherman80 — APPROVED** [review]

File: spec/application_controller_spec.rb [hunk 1]
```diff
@@ -89,9 +89,8 @@ def app
     expect(body[:message]).to include('Welcome to the Yelp Microservice!')
   end
 
-  # Test is causing issues with Travis - further research necessary
   xit 'can hit the utilities/electricity endpoint' do
-    location = 80_211
+    location = 80211
```
Comment (Stearnzy) [inline spec/application_controller_spec.rb:93]: Will these tests be unskipped eventually or are we keeping them this way?

**Review by Stearnzy — COMMENTED** [review]
Should we not merge this until the business show endpoint is put back in?

File: README.md [hunk 2]
```diff
@@ -25,11 +25,6 @@ Response:
 }
 ```
 
-### Business ID Requests (Show):
```
Comment (Callbritton) [inline README.md:28]: Looks like we will be needing that businesses/:id route after all!

File: README.md [hunk 3]
```diff
@@ -84,3 +79,30 @@ Example Response for `GET '80211/utilities/electricity'`
...
+### Local Events
+```
+GET ':location/events'
+```
+Example Response for `GET '80211/events'`
+```
+{
+    "events": [
+        {
+            "id": "denver-upon-closer-reflection-opening-reception-and-exhibition",
```
Comment (Callbritton) [inline README.md:92]: Great work handrolling this!

File: spec/application_controller_spec.rb (hunk 1 above)
Comment (jakeheft) [reply spec/application_controller_spec.rb:93]: We are keeping them this way until we figure out what is causing Travis to fail

**Pull request merged**